diff --git a/index.d.ts b/index.d.ts
index 4c664f51b79e2ac9ee670aac880334648a12e19c..3573413912755eb5c3c15cd4eeb42be0dda669af 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -120,6 +120,10 @@ declare namespace Arborist {
         protected constructor(options: never);
         /** The name of this node's folder in `node_modules`. */
         name: string;
+
+        // Added arborist v6
+        get version(): string;
+
         /**
          * Physical parent node in the tree.  The package in whose `node_modules`
          * folder this package lives.  Null if node is top of tree.
@@ -206,9 +210,9 @@ declare namespace Arborist {
          * Edges in the dependency graph indicating nodes that this node depends
          * on, which resolve its dependencies.
          */
-        edgesOut: Edge[];
+        edgesOut: Map<string, Edge>;
         /** Edges in the dependency graph indicating nodes that depend on this node. */
-        edgesIn: Edge[];
+        edgesIn: Set<Edge>;
 
         /** True if this package is not required by any other for any reason.  False for top of tree. */
         extraneous: boolean;