diff --git a/index.d.ts b/index.d.ts
index 4c664f51b79e2ac9ee670aac880334648a12e19c..02ae73d1c51b46ace81249e9284b0cc55d574220 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -134,6 +134,8 @@ declare namespace Arborist {
          */
         meta?: Shrinkwrap;
 
+        get version(): string;
+
         /** Map of packages located in the node's `node_modules` folder. */
         children: Map<string, Node | Link>;
 
@@ -206,9 +208,9 @@ declare namespace Arborist {
          * Edges in the dependency graph indicating nodes that this node depends
          * on, which resolve its dependencies.
          */
-        edgesOut: Edge[];
+        edgesOut: Map<string, Edge>;
         /** Edges in the dependency graph indicating nodes that depend on this node. */
-        edgesIn: Edge[];
+        edgesIn: Set<Edge>;
 
         /** True if this package is not required by any other for any reason.  False for top of tree. */
         extraneous: boolean;
@@ -265,7 +267,7 @@ declare namespace Arborist {
          */
         spec: string;
         /** Automatically set to the node in the tree that matches the `name` field. */
-        to: Node;
+        to: Node | null;
         /** True if `edge.to` satisfies the specifier. */
         valid: boolean;
         /**