diff --git a/index.d.ts b/index.d.ts
index 4c664f51b79e2ac9ee670aac880334648a12e19c..78b54315bf989d5e021b0842b1c69339ee108684 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -206,9 +206,9 @@ declare namespace Arborist {
          * Edges in the dependency graph indicating nodes that this node depends
          * on, which resolve its dependencies.
          */
-        edgesOut: Edge[];
+        edgesOut: Set<Edge>;
         /** Edges in the dependency graph indicating nodes that depend on this node. */
-        edgesIn: Edge[];
+        edgesIn: Set<Edge>;
 
         /** True if this package is not required by any other for any reason.  False for top of tree. */
         extraneous: boolean;